eldritch/events.py

```diff
@@ -174,9 +174,10 @@ def resolve(self, state):
       self.done = True
       return True
     assert self.dest in self.character.place.connections
-    self.character.place = self.dest
-    self.character.movement_points -= 1
-    self.character.explored = False
+    if not self.dest.closed:
```

Comment:
Interesting to note that if a person tries to move past a closed location, they'll get an assertionerror anyway when the next MoveOne asserts that the two locations are connected.

Not a problem, I think it's fine. The UI shouldn't let people move to or past closed locations anyway.